src/main/java/de/tum/in/www1/artemis/domain/quiz/QuizQuestion.java

```diff
@@ -261,8 +261,8 @@ public void filterForStatisticWebsocket() {
      */
     @JsonIgnore
     public Boolean isValid() {
-        // check title
-        return getTitle() != null && !getTitle().equals("");
+        // check title and score
+        return getTitle() != null && !getTitle().equals("") && getScore() > 0;
```

Comment:
What about multiple whitespaces. You should maybe trim the string before you do the validation. Could be also a problem in the frontend. Please check this! 💯

Reply:
getScore() returns an Integer, therefore no trimming/parsing is needed at this place.


_I just tested the behaviour of the server when entering letters and whitespace, but since the input is a number, everything behaves as expected._
![image](https://user-images.githubusercontent.com/63976129/82737806-dd4e2380-9d33-11ea-8b19-d2bb50cbfc69.png)

Reply:
I do agree on that, but since the related issue #1393 refers to the non-positive number score I didn't want to change too much here - only the things related to the issue.

I think it would make more sense to create a new issue regarding this.